projects/sandbox/datagen/datagen/scripts/background.py

```diff
@@ -235,11 +237,11 @@ def deploy(
     ifos: List[str],
     sample_rate: float,
     channel: str,
-    state_flag: str,
     datadir: Path,
     logdir: Path,
     accounting_group: str,
     accounting_group_user: str,
+    state_flag: Optional[str] = None,
```

Comment:
I'll suggest also changing this to be `"DATA"`, as well as the argument in `deploy`